components/select/select.tsx

```diff
@@ -168,12 +175,12 @@ const Select: React.FC<React.PropsWithChildren<SelectProps>> = ({
             width: ${width};
             overflow: hidden;
             transition: border 0.2s ease 0s, color 0.2s ease-out 0s, box-shadow 0.2s ease 0s;
-            border: 1px solid ${theme.palette.border};
+            border: ${isText ? 'none' : `1px solid ${theme.palette.cNeutral3}`};
```

Comment:
Better put this kind of logic into a separate style file

Reply:
done